Makefile

```diff
@@ -176,6 +176,10 @@ cypress-local:
 cypress-local-shell:
 	aws-vault exec moj-lpa-dev -- docker run -it -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e "CYPRESS_baseUrl=https://localhost:7002" --entrypoint bash --network="host" -v `pwd`/cypress:/app/cypress cypress:latest
 
+.PHONY: cypress-gui-npm
```

Comment:
Devil's advocate Q - does putting this in the Makefile really add any value?  Cos make cypress-gui-npm is no less to type than cypress open, but the latter makes you think about what you're doing more, and works anywhere that cypress is installed.... ;)

Reply:
it is a good point - i almost didn't put this in. more for consistency.

Reply:
the original Readme i did listed `cypress open` as the way to run it, but then thought might as well add it. it's neither here nor there as you say in terms of keystrokes. the make does specify the folder to run, which means it auto picks up the cypress folder correctly.

Reply:
bit of rejigging done, good for a re-review also used MD lint locally and fixed a few things.